templates/default/livy.conf.erb

```diff
@@ -52,4 +52,8 @@ livy.server.recovery.state-store=filesystem
 
 livy.server.recovery.state-store.url = file://<%= node['livy']['base_dir'] %>/logs
 
-livy.repl.enableHiveContext = true
\ No newline at end of file
+livy.repl.enableHiveContext = true
+
+#RPC size properties, for large installations it might have to be further increased. Set default to 256MB
+livy.rsc.rpc.max.size= <%= node['livy']['rsc']['rpc']['max']['size'] %>
+livy.rpc.max.size= <%= node['livy']['rsc']['rpc']['max']['size'] %>
```

Comment:
I guess you meant `livy.rpc.max.size= <%= node['livy']['rpc']['max']['size'] %>`

Reply:
I need a new pair of eyes